packages/proxy/lib/http/response-middleware.ts

```diff
@@ -225,6 +225,37 @@ const SetInjectionLevel: ResponseMiddleware = function () {
   this.next()
 }
 
+// https://github.com/cypress-io/cypress/issues/6480
+const MaybeStripDocumentDomainFeaturePolicy: ResponseMiddleware = function () {
+  // "autoplay *; document-domain 'none'" => { autoplay: "*", "document-domain": "'none'" }
+  const parseFeaturePolicy = (policy: string): any => {
+    const pairs = policy.split('; ').map((directive) => directive.split(' '))
+
+    return _.fromPairs(pairs)
+  }
+
+  // { autoplay: "*", "document-domain": "'none'" } => "autoplay *; document-domain 'none'"
+  const stringifyFeaturePolicy = (policy: any): string => {
+    const pairs = _.toPairs(policy)
+
+    return pairs.map((directive) => directive.join(' ')).join('; ')
+  }
```

Comment:
nit: other functions in this file are declared at the top and then used later on in the middlewares, you may want to follow that pattern just for consistency's sake. not a big deal